### DISCUSSION TITLE
Web 3.0 Game Changer

### DISCUSSION THREAD

**Discussion by ahmedjsofficial** [community/community#19312] [GitHub Education]
#  Web3.0 Game Changer
📢📣Web3.0 is the game changer in the modern world. It has becoming the most popular and advanced technology, On based of Decentralized APPs (DAPPS) and Blockchain. In this one video you will learn about the Thirdweb. You will everything that you needs for you next level jobs and projects. You will develop your own NFTs Minter by using ERC721 Smart Contracts (NFTs Drop).You will learn the NEXT.JS from scratch and best react advanced practices. You will learn NEXT.JS Dynamic Routing, Incredible Techniques and much more.

🔴 MINT NFTS ON ETHEREUM BLOCKCHAIN 🚀🚀🚀🚀
Subscribe if you wants to become a Smart Contract Developer in this modern world.

Click this link to become a smart contract developer: https://www.youtube.com/watch?v=sEUT0Wn3UPc&t=2153s

**Comment by samus-aran** [comment]
Hi @jsstackdevelopers , we’re glad you’re here 😀
Thanks for sharing your enthusiasm with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
